SparkleSpin/Views/LightButton.swift

```diff
@@ -0,0 +1,72 @@
+//
+//  LightButton.swift
+//  SparkleSpin
+//
+//  Created by Britney Smith on 3/5/19.
+//  Copyright © 2019 Britney Smith. All rights reserved.
+//
+
+import UIKit
+
+@IBDesignable
+class LightButton: UIButton {
+    
+
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        styleButton()
+    }
+    
+    required init?(coder aDecoder: NSCoder) {
+        super.init(coder: aDecoder)
+        styleButton()
+    }
+    
+    override var isHighlighted: Bool {
+        didSet {
+            if isHighlighted {
+                isHighlighted = false
+                self.setTitleColor(ThemeColor.Light.accentColorTwo, for: .normal)
+            } else {
+                self.setTitleColor(ThemeColor.Light.secondaryColor, for: .normal)
+            }
+            
+        }
+    }
+    
+    
+    private func styleButton() {
+        
+        backgroundColor = ThemeColor.Light.primaryColor
+        layer.cornerRadius = 8
+        layer.shadowColor = ThemeColor.Light.secondaryColor?.cgColor
+        layer.shadowOffset = CGSize(width: 1.5, height: 2.0)
+        layer.shadowOpacity = 0.9
+        layer.shadowRadius = 2.0
+        layer.masksToBounds = false
+        
+        
```

Comment:
You might want to consider using a [linter](https://github.com/realm/SwiftLint) to help you automatically detect multiple empty lines. You don't have to do this but I think it would be another fun thing to learn and that you could benefit from it.